src/constants/config.js

```diff
@@ -5,13 +5,15 @@ const envConfig = {
     workflowUrl: 'https://pfe-preview.zooniverse.org/projects/rafe-dot-lafrance/notes-from-nature-on-staging/classify?reload=1&',
     panoptesReturnUrl: 'http://localhost:3000/',
     panoptesAppId: '24ad5676d5d25c6aa850dc5d5f63ec8c03dbc7ae113b6442b8571fce6c5b974c',
+    apiUrl: 'https://panoptes-staging.zooniverse.org/api/',
```

Comment:
are the new `apiUrl` properties on the `staging` and `production` objects related to this PR or from unrelated work?